.vscode/launch.json

```diff
@@ -16,7 +41,10 @@
         "${workspaceFolder}/readme.md",
         "--extensionDevelopmentPath=${workspaceFolder}"
       ],
-      "skipFiles": ["<node_internals>/**", "**/node_modules/vscode-*/**"],
+      "skipFiles": [
+        "<node_internals>/**",
+        "**/node_modules/vscode-*/**"
+      ],
```

Comment:
All changes in this file are unrelated. Please revert them.

Reply:
I'm happy to remove this, it did help me with testing a lot though. It gives a developer the possibility to test the extension in sandbox mode with breakpoints.
Are you sure this is not something you'd want?